paasta_tools/cli/cmds/wait_for_deployment.py

```diff
@@ -81,6 +81,13 @@ def add_subparser(subparsers):
         required=True,
         type=validate_short_git_sha,
     )
+    list_parser.add_argument(
```

Comment:
Do we also need to change `paasta_wait_for_deployment` in this file (and possibly the validation stuff)?

Reply:
I was originally trying to target just mark_for_deployment.py with this PR to minimize changes, but I guess it's small enough I'll throw that in here too.